### ISSUE TITLE
Spezialisierung

### ISSUE THREAD

**Issue by jwssnr** [jwssnr/feeef2#12]
Ein Detail zur Ausbildung wurde in #10 schon eingeworfen: Nämlich das Szenario eines Basisjahres mit anschliessender Spezialisierung. Ich finde das Thema wichtig, wenn auch noch etwas früh im Prozess. Aber ich möchte es hier deponieren.

Generell bin ich nicht so ein Freund der Spezialisierungen innerhalb einer Ausbildung. Das mag bei einigen Berufen Sinn ergeben. Aber mich dünkt eine solide, breit gestützte, praxisnahe Grundbildung wichtig, die sich nicht in Oberflächlichkeiten verliert, bzw. versucht eierlegende Wollmilchschweine zu produzieren. Spezialisieren soll man sich nachher. Oder wie @dimitrisurber in sagt, im Lehrbetrieb. Aus einem Grund heisst es ja Grundbildung.

Zudem haben wir hier schon eine attraktive Mischung zwischen Technik und Kreation.

**Comment by signalwerk** [comment]
@jwssnr ich stimme dir zu. ich komme aus einem Berufsbild (Polygraf), welches damals die Spezialisierung «Bild» oder «Text» hatte. Ich war und bin der Meinung, dass dies nur gemacht ist um Lehrbetrieben das Gefühl zu geben, dass Sie das richtige Ausbilden. Widerspiegelt aber nicht die Realität einer beruflichen Karriere. Grundbildung zuerst. Spezialisierung später.
